### PR TITLE
Travels UI improvements

### PR DESCRIPTION
**Pull request by lucasponce**
Some minor UI improvements that may improve the use of the Travels demo:

image

- Classic screen resolution (I'm looking at you @jmazzitelli) are now better supported.
- Improvements map labels (now cities have # of requests).
- Labels for controls, to better explain % of control and results in the charts.
- Minor wording and enhancements in the UI code.

Only UI work, no changes on deployments or configuration

In general, now it's easier to show Travels UI and Kiali without resize browser or adjust resolutions.

### CODE REVIEW

**Comment by lucasponce** [comment]
Note, that once that is reviewed, new images would need to be generated.

In the future, we could use some Github Action to automate this step, but today it's manual.

**Review by xeviknal — APPROVED** [review]
LGTM! reviewed, installed and deployed. Everything looks on place.

**Review by jshaughn — APPROVED** [review]
Everything looks good to me!  One tiny comment about formatting.

File: travels/travel_control/html/js/travel_control.js [hunk 1]
```diff
@@ -14,27 +14,36 @@ const buildPortalsForm = (portals) => {
         form.id = portal.name;
         form.className = 'left-form';
         errorContent = portal.status.error ? ' - Failed to connect' : '';
+        const ratioPerSec = portal.settings.request_ratio > 0 ? (portal.settings.request_ratio/100) * 1.33 : 0.06;
         form.innerHTML = `
-            <h2>${portal.name} Settings <span id="${portal.name}_error" class="error">${errorContent}</span></h2>
+            <h3>${portal.name} <span id="${portal.name}_error" class="error">${errorContent}</span></h3>
             <div class="left-form-group">
-                <label for="${portal.name}_ratio">Requests Ratio</label>
+                <label>Ratio</label>
```
[the remark this answers is not in the document]
Comment (lucasponce) [reply travels/travel_control/html/js/travel_control.js]: Thanks !
I did some standard formatting from IDE but as this is the demos repo, I won't introduce any strong "formatting" tool that may add more noise.

**Pull request merged**